src/Components/Reports/Common/ExecutiveFirstPage.js

```diff
@@ -108,6 +108,15 @@ export const ExecutiveFirstPage = ({ data }) => {
 
         <Text style={styles.execHeading}>Breakdown of registered systems</Text>
 
+        {
+            nonPSICount > 0
+             && <Text>{
+                 /* eslint-disable-next-line max-len */
+                 `${psiEnabledCount} systems out of a total of ${totalCount} systems have Kernel Pressure Stall Information enabled. You could get better suggestions for ${nonPSICount} systems if you enabled Pressure Stall Information. Check the documentation on how to enable PSI on versions RHEL 8 and newer.`
```

Comment:
Should we also include plural/singular here? As count can be 1 -`${psiEnabledCount} systems `

Reply:
good catch @r14chandra , fixed it and pushed the changes to PR. Thanks!